tests/trace/type_handlers/Video/README.md

```diff
@@ -0,0 +1,37 @@
+# Video Type Handler Tests
```

Comment:
Any reason to capitalize the directory name?

Reply:
Convention, all the other type handlers have capital names for their directories. I guess because it refers to a type 'Video' and types are capitalized